### ISSUE TITLE
Generated node name can not pass the name validation rules

### ISSUE THREAD

**Issue by pepyakin** [paritytech/substrate#1590]
```
./target/release/substratenode --dev
Error: Invalid node name 'enthusiastic-representative-3454'. Reason: Node name too long. If unsure, use none.
```

**Comment by gavofyork** [comment]
@pepyakin all issues should be given milestones at creation. If you're unsure just give it `As-and-when`.

**Comment by pepyakin** [comment]
Understood. This one seems to be fixed by #1596. Closing...
